lib/src/rules/cascade_invocations.dart

```diff
@@ -51,6 +52,14 @@ someReference
 
 ''';
 
+Element _findElement(SimpleIdentifier identifier) {
+  if (identifier.bestElement is PropertyAccessorElement) {
```

Comment:
I can't because I don't know the reason, I just ran the test and it failed, so I need to dig deeper into this and gain some knowledge on the structure of the tree.